### ISSUE TITLE
Initial merged branch from main creates chore PR, subsequent branches off main do not. 

### ISSUE THREAD

**Issue by jaysonpotter** [googleapis/release-please-action#998]
#### TLDR;
https://youtu.be/-xkt2RKCaEQ - 2:43
or
Created two branches off of the main branch, both with commit messages following the conventional commit convention, both with changes different from the other. Merging the first will create a chore PR, the second will not. 

https://github.com/jaysonpotter/github-actions-tests - fresh repo that demonstrates the issue. 

#### Environment details

  - Programming language: NA
  - OS: NA
  - Language runtime version: NA
  - Package version: 4

#### Steps to reproduce

  1. Create a new repo containing a package.json and .gitgub/workflows/release-please.yml config.
  2. Check "Allow GitHub Actions to create and approve pull requests" in Settings > Actions > General > Workflow permissions
  3. Create a branch off of default branch (main, master, etc); Update-1
  4. Create a branch off of default branch (main, master, etc); Update-2
  5. Add a change to Update-1. In my demo I add an index.html
  6. Add a change to Update-2. In my demo I add a README.md
  7. Create PR to merge Update-2 to default branch. 
  8. Merge Update-2 to default branch.
  9. Go to Actions and see release-please run and create a chore PR.
  10. Go to Pull requests and merge chore PR. 
  11. Create PR to merge Update-1 to default branch.
  12. Merge Update-1 to default branch.
  13. Go to Actions and view the details of the release-please workflow and you'll see that the Update-1 PR "commit could not be parsed" and no chore PR is generated. 

#### release-please.yml
`
on:
  push:
    branches:
      - main

permissions:
  contents: write
  pull-requests: write

name: release-please

jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: googleapis/release-please-action@v4
        with:
          release-type: node
`

Feel free to see any details in the repo I created to demonstrate this issue at https://github.com/jaysonpotter/github-actions-tests

Thank you for your time and help!

**Comment by jaysonpotter** [comment]
Bueller?
